cmd/kops/set.go

```diff
@@ -33,7 +33,8 @@ var (
 
 	setExample = templates.Examples(i18n.T(`
     # Set cluster to run kubernetes version 1.17.0
-    kops set cluster k8s-cluster.example.com spec.kubernetesVersion=1.17.0
+	kops set cluster k8s-cluster.example.com spec.kubernetesVersion=1.17.0
+	kops set instancegroup k8s-cluster.example.com spec.maxSize=4
```

Comment:
this will require the cli docs to be regenerated:

```suggestion
	kops set instancegroup --name k8s-cluster.example.com nodes spec.maxSize=4
```

Reply:
Good catch, thanks. I had this changed locally, but missed including it in the branch. Should be fixed now.